.buildkite/pipeline.yml

```diff
@@ -48,8 +64,6 @@ steps:
         allow_failure: false
       - step: unit-tests-linux
         allow_failure: false
-      - step: unit-tests-windows
-        allow_failure: false
```

Comment:
Ok, no need to wait for windows unit tests step if linux unit tests are successful. :+1: 

What I would suggest is to move the `wait` step before the windows unit test step

```yaml
  - label: ":go: :linux: Run unit tests"
    key: unit-tests-linux
    command: "make test-go-ci"
    artifact_paths:
      - "build/test-results/*.xml"
      - "build/test-coverage/*.xml"
    agents:
      image: "${LINUX_AGENT_IMAGE}"
      cpu: "8"
      memory: "4G"

  - wait: ~
    continue_on_failure: true

  - label: ":go: :windows: Run unit tests"
    key: unit-tests-windows
    command: ".buildkite/scripts/unit_tests_windows.ps1"
    agents:
      provider: "gcp"
      image: "${WINDOWS_AGENT_IMAGE}"
    artifact_paths:
      - "TEST-unit-windows.xml"

  - label: ":pipeline: Trigger Integration tests"
    command: ".buildkite/pipeline.trigger.integration.tests.sh | buildkite-agent pipeline upload"
    depends_on:
      - step: check-static
        allow_failure: false
      - step: unit-tests-linux
        allow_failure: false
```

Mainly for the UI view because the trigger integration tests will run even windows steps are still running

![buildkite steps](https://github.com/elastic/elastic-package/assets/5330827/4ef4479d-a668-4101-bce2-6a0940231848)

Reply:
Ok, I see it has been moved the steps, and the step to run stack in windows is running at the same time as the other integration test.

Ignore this comment.